gnome-initial-setup/pages/location/gis-location-page.c

```diff
@@ -812,6 +812,9 @@ gis_location_page_init (GisLocationPage *page)
 void
 gis_prepare_location_page (GisDriver *driver)
 {
+  if (gis_driver_is_live_session (driver))
+    return;
```

Comment:
We definitely want to hide it. If feasible, I would like to *also* avoid changing the hardware clock. But, if booting a current Endless OS live USB on a Windows machine (with a non-UTC timezone) doesn't mess with the clock, just hiding the page is probably fine. (I don't have one to test on ☹)